### ISSUE TITLE
Improve onboarding experience

### ISSUE THREAD

**Issue by finnbar** [finnbar/tablebot#14]
While the issues are in-depth and the code documentation good, there is not a lot to help a Haskell programmer get to grips with writing code for the system. A list of useful tricks would be very helpful.

- [ ] Explanation of `DatabaseDiscord`, including discussion of `liftIO`.
- [ ] More in-depth README.

**Comment by L0neGamer** [comment]
When adding a plugin, it has to be exported in the same file as the plugin, and imported in Plugins.hs, and exported by Plugins.hs, and imported by Main.hs, and included in the list in Main.hs

**Comment by finnbar** [comment]
Yeah, I think there's work to be done re: that. It might be best to have every plugin imported by default and then turn them off at runtime - so instead of `runTablebot token prefix path plugins`, omit the last argument by having it default to every plugin (so it sets up every plugin correctly) and then allow for unwanted plugins to be disabled.

**Comment by finnbar** [comment]
Next job on this - a tutorial on ping, ping with what time it is, ping with database saving of number of pings, ping with optional user argument of how many pings to do.
